src/rdcrc32.h

```diff
@@ -70,7 +70,7 @@ rd_crc32_t rd_crc32_reflect(rd_crc32_t data, size_t data_len);
  *****************************************************************************/
 static __inline rd_crc32_t rd_crc32_init(void)
 {
-#if WITH_ZLIB
+#if WITH_ZLIB && !defined(_MSC_VER)
```

Comment:
Make it work with all possible inputs. It currently silently gives you the wrong crc32 if you give it a buffer that's >2 GB.

But I suppose that's not going to happen, I'll change it to an assert < 2GB and use zlib.